ietf/templates/api/index.html

```diff
@@ -25,7 +25,7 @@ <h2 id="framework">Framework API</h2>
         </p>
         <p>
             The API top endpoint is at
-            <a href="https://datatracker.ietf.org/api/v1/">https://datatracker.ietf.org/api/v1/</a>.
+            <a href="{{ settings.IDTRACKER_BASE_URL }}/api/v1/">{{ settings.IDTRACKER_BASE_URL }}/api/v1/</a>.
```

Comment:
While these _could_ be shifted to use reverse, the benefit from doing so is so slight that doing the work to find the computed view names under tastypie isn't worth it.